### PR TITLE
LG-2008 dynamic react example

### PR DESCRIPTION
**Pull request by TheSonOfThomp**

### CODE REVIEW

File: components/pages/example/LiveExample.tsx [hunk 1]
```diff
@@ -79,7 +180,9 @@ export const LiveExample = ({
       .then(module => {
         if (module) {
           const { default: meta, ...stories } = module;
-          const defaultStoryName = meta?.parameters?.default;
+
+          const defaultStoryName =
+            meta.parameters?.default ?? Object.keys(stories)[0];
```
Comment (spark33) [inline components/pages/example/LiveExample.tsx:185]: Is there a chance the `Object.keys(stories)[0]` fails in cases where `stories` is empty?

File: components/pages/example/LiveExample.tsx [hunk 2]
```diff
@@ -110,40 +213,80 @@ export const LiveExample = ({
...
       darkMode={darkMode}
       className={css`
         margin-block: 2em;
       `}
     >
-      <div className={storyWrapperStyle}>
-        {StoryFn ? <StoryFn {...knobValues} /> : <H2>No example found 🕵️</H2>}
+      <div className={liveExampleWrapperStyle}>
+        <div className={storyWrapperStyle}>
+          {StoryFn ? <StoryFn {...knobValues} /> : <H2>No example found 🕵️</H2>}
```
Comment (spark33) [inline components/pages/example/LiveExample.tsx:257]: 🕵️

File: components/pages/example/LiveExample.tsx [hunk 3]
```diff
@@ -110,40 +213,80 @@ export const LiveExample = ({
...
+        <Transition in={showCode} timeout={200}>
+          {state => (
+            <div
+              className={cx(
+                codeExampleWrapperStyle,
+                codeWrapperStateStyle[state],
+              )}
+            >
+              <Code className={codeStyle} darkMode={darkMode} language="js">
+                {storyCode ?? 'No code found'}
```
Comment (spark33) [inline components/pages/example/LiveExample.tsx:268]: 🕵️? 😂

Comment (spark33) [reply components/pages/example/LiveExample.tsx:268]: Or actually, should the `No code found` text just be a Typography component outside the Code component?

File: components/pages/example/LiveExample.tsx (hunk 1 above)
Comment (TheSonOfThomp) [reply components/pages/example/LiveExample.tsx:185]: It gets caught in the `try/catch` and renders "No Story Found". Check out `palette`

**Review by spark33 — APPROVED** [review]

File: components/pages/example/LiveExample.tsx (hunk 3 above)
Comment (TheSonOfThomp) [reply components/pages/example/LiveExample.tsx:268]: I think it makes sense to have the text inside the code component 
Screen Shot 2022-10-20 at 1 21 40 PM

**Pull request merged**